### PR TITLE
Updated strings from Crowdin 20210727

### LINKED ISSUE #9319: Resources: Allow strings containing '&'
We need either need

* a lint check to ensure that strings do not contain an `&amp`. A user should use `&` in a CDATA block instead
   * I'd personally be happy with this to mark the issue as fixed
* or fix the localization script
  * Needs thorough testing, I wouldn't consider this to be a good first issue as it involves CrowdIn API access and we don't have a test framework yet
  * See: https://github.com/ankidroid/Anki-Android/pull/9318#issuecomment-887631641

See a working example:

https://github.com/ankidroid/Anki-Android/blob/897f932b6f673327497246894dd2362d1917d287/AnkiDroid/src/main/res/values/02-strings.xml#L260

_Originally posted by @david-allison-1 in https://github.com/ankidroid/Anki-Android/issues/9318#issuecomment-887633139_

### PR DESCRIPTION
**Pull request by mikehardy**
- tool commit to fix path issue
- tool commit to fix #9319 
- updated translations running updated tool

### CODE REVIEW

File: AnkiDroid/src/main/res/values-af/10-preferences.xml [hunk 1]
```diff
@@ -312,4 +312,14 @@
     <!-- Open Changelog-->
     <string name="open_changelog">Open Changelog</string>
     <string name="binding_add_gesture">Add gesture</string>
+    <string name="binding_add_key">Add key</string>
+    <string name="key_picker_default_press_key">Press a key</string>
+    <!-- Select card side -->
+    <string name="card_side_selection_title">Select card side</string>
+    <string name="card_side_question">Question</string>
+    <string name="card_side_answer">Answer</string>
+    <string name="card_side_both">Question & Answer</string>
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/values-af/10-preferences.xml:321]: These are a problem:

```

ERROR:/home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/res/values-ml/10-preferences.xml:320:45: Resource and asset merger: The entity name must immediately follow the '&' in the entity reference.
```

**Comment by david-allison-1** [comment]
Unit test failure. Likely caused by 

This was input as 

> `<string name="card_side_both">Question &amp; Answer</string>`

https://github.com/ankidroid/Anki-Android/pull/9289/files#diff-019694600ddf48c04a96ea5326a284af3b523cb16e8dc07af2120e791ec232fcR312

But came back as 

> `<string name="card_side_both">Question & Answer</string>`

**Comment by david-allison-1** [comment]
When editing in CrowdIn, it appears as `&amp;`, which is correct:

https://crowdin.com/translate/ankidroid/8173/en-ar?filter=basic&value=0

**Comment by david-allison-1** [comment]
Likely:

https://github.com/ankidroid/Anki-Android/blob/56e5194b2dd4c7c1d668296acdbc3bfccd4c601e/tools/update-localizations.py#L71

**Comment by david-allison-1** [comment]
We need either need a lint check to ensure that strings do not contain an `&` unless they're in a `CDATA` block (or fix the script)

See:

https://github.com/ankidroid/Anki-Android/blob/897f932b6f673327497246894dd2362d1917d287/AnkiDroid/src/main/res/values/02-strings.xml#L260

File: AnkiDroid/src/main/res/values-af/10-preferences.xml (hunk 1 above)
Comment (mikehardy) [reply AnkiDroid/src/main/res/values-af/10-preferences.xml]: Got lucky! Removing that single replacement line in the script seemed valid to me, pushed the results here

**Comment by mikehardy** [comment]
@david-allison-1 I apologize this is a bit of a 3-in-1, I could untangle to separate PRs but I admit to not caring as much on maintenance / tooling-related items in non-primary languages here.

This should *definitely* be squash-merged though, otherwise the commit history will have a lint-failing commit

**Comment by david-allison-1** [comment]
Sorry to be a bother. Could you rebase this into two squashed commits so we can rebase this onto master, rather than squash.

I'm happy with the overall change, but having the "tool change" PR as a separate commit (first) is a strong desire, otherwise it will become lost in the noise of the standard translation sync.

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
Adding the review for bookkeeping, see comment on rebasing

**Comment by mikehardy** [comment]
no bother - my git-fu can easily handle that - will get it first thing tomorrow morning (my time)

**Comment by mikehardy** [comment]
Had something unexpected come up all morning but finally mashed the changes around and have a new batch of commits, what do you think @david-allison-1  ?

**Review by david-allison-1 — APPROVED** [review]
LGTM, never any time pressure here!

**Pull request merged**